From 8b090dec49da1d238ce9e6be1b4b9627707a1c2f Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Wed, 27 Oct 2021 11:59:52 -0400
Subject: [PATCH] iproute.spec: Modify Release field for StarlingX

This commit modifies the Release field of the iproute package's spec
file for StarlingX.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 SPECS/iproute.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SPECS/iproute.spec b/SPECS/iproute.spec
index 1b5b5a719fdc..20869a9defbd 100644
--- a/SPECS/iproute.spec
+++ b/SPECS/iproute.spec
@@ -1,7 +1,7 @@
 Summary:            Advanced IP routing and network device configuration tools
 Name:               iproute
 Version:            5.12.0
-Release:            4%{?dist}%{?buildid}
+Release:            4%{?_tis_dist}.%{tis_patch_ver}%{?buildid}
 Group:              Applications/System
 URL:                http://kernel.org/pub/linux/utils/net/%{name}2/
 Source0:            http://kernel.org/pub/linux/utils/net/%{name}2/%{name}2-%{version}.tar.xz
-- 
2.29.2

